packages/manager/src/queries/placementGroups.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  assignVmsToPlacementGroup,
+  createPlacementGroup,
+  deletePlacementGroup,
+  getPlacementGroup,
+  getPlacementGroups,
+  unassignVmsFromPlacementGroup,
+  updatePlacementGroup,
+} from '@linode/api-v4';
+import {
+  APIError,
+  Filter,
+  Params,
+  ResourcePage,
+} from '@linode/api-v4/lib/types';
+import { useMutation, useQuery, useQueryClient } from 'react-query';
+
+import { queryKey as PROFILE_QUERY_KEY } from './profile';
+
+import type {
+  CreatePlacementGroupPayload,
+  PlacementGroup,
+  UpdatePlacementGroupPayload,
+} from '@linode/api-v4';
+
+export const queryKey = 'placement-groups';
+
+export const usePlacementGroupsQuery = (
+  params: Params,
+  filter: Filter,
+  enabled: boolean = true
+) =>
+  useQuery<ResourcePage<PlacementGroup>, APIError[]>({
+    enabled,
+    keepPreviousData: true,
+    queryFn: () => getPlacementGroups(params, filter),
+    queryKey: [queryKey, 'paginated', params, filter],
+  });
+
+export const usePlacementGroupQuery = (
+  placementGroupId: number,
+  enabled: boolean = true
+) => {
+  return useQuery<PlacementGroup, APIError[]>({
+    enabled,
+    queryFn: () => getPlacementGroup(placementGroupId),
+    queryKey: [queryKey, 'placement-group', placementGroupId],
+  });
+};
+
+export const useCreatePlacementGroup = () => {
+  const queryClient = useQueryClient();
+
+  return useMutation<PlacementGroup, APIError[], CreatePlacementGroupPayload>({
+    mutationFn: (data) => createPlacementGroup(data),
```

Comment:
Optional: we could keep the shorthand for this

```suggestion
    mutationFn: createPlacementGroup,
```

Reply:
oops yes, fixed